### ISSUE TITLE
[Feature Request]: Ability to download multiple games all at once.

### ISSUE THREAD

**Issue by nicktheone** [JMBeresford/retrom#192]
My request stems from the fact that because of Retrom ROMs folder structure (emulation system folder -> game folder -> ROM instead of just emulation system folde -> all of the ROMs) if I want to add more than one game (or if I'm setting up a new emulation device from scratch) I need to either go through each folder and grab each ROM one by one using a file browser or I can use Retrom web interface to download each ROM one by one.

Since I'm sure there's a reason for the folder structure used by Retrom I'd like to at least be able to select multiple games or even whole systems and download a zip archive with all the games in it (or automatically start the download for all the selected games).

**Comment by JMBeresford** [comment]
@nicktheone Firstly, if I understand you correctly here:

```
just emulation system folde -> all of the ROMs
```

Retrom does support this folder structure, check out [this part of the README](https://github.com/JMBeresford/retrom/tree/retrom-v0.4.5?tab=readme-ov-file#single-file-games) for more details.

However I will indeed be implementing bulk actions such as you describe, eventually.

**Comment by nicktheone** [comment]
@JMBeresford Yes, that was what I meant. I've read the part of the documentation that you linked but you've also specifically said that folder structure isn't recommended. That's why I went with the _one folder, one game_ structure you recommended in the documentation.

It's not really a problem for me, just a minor nitpick that I'd like eventually to see corrected.
